roles/tendrl-storage-node/handlers/main.yml

```diff
@@ -5,3 +5,8 @@
   service:
     name=tendrl-node-agent
     state=restarted
+
+- name: restart firewalld
```

Comment:
Are we using this anywhere?

Reply:
I am using this to restart firewalld after enabling port for grafana server
in roles/tendrl-server/tasks/firewalld.yml

Reply:
Use reload instead of restart

Reply:
Sure, however i have seen few times that reload does not works. But restart works fine also reload after restart works fine.

Reply:
Reload is not working only restart is working fine. What is your opinion about keeping this as restart only for now and will enhance later.

Reply:
Hmm, `systemctl reload firewalld` does reload permanent rules and makes them active:

```
# firewall-cmd --add-service=http --permanent
# firewall-cmd --list-all --permanent # this shows the http service
# firewall-cmd --list-all # this doesn't show http service
# systemctl reload firewalld
# firewall-cmd --list-all # now http service is shown here
```